src/sweeps/search/bayes.py

```diff
@@ -451,83 +450,55 @@ def _construct_gp_data(
     return params, sample_X, current_X, y
 
 
-def bayes_search_next_run(
-    runs: List[SweepRun],
-    config: Union[dict, SweepConfig],
-    validate: bool = False,
-    minimum_improvement: floating = 0.1,
-) -> SweepRun:
-    """Suggest runs using Bayesian optimization.
-
-    >>> suggestion = bayes_search_next_run([], {
-    ...    'method': 'bayes',
-    ...    'parameters': {'a': {'min': 1., 'max': 2.}},
-    ...    'metric': {'name': 'loss', 'goal': 'maximize'}
-    ... })
-
-    Args:
-        runs: The runs in the sweep.
-        config: The sweep's config.
-        minimum_improvement: The minimium improvement to optimize for. Higher means take more exploratory risks.
-        validate: Whether to validate `sweep_config` against the SweepConfig JSONschema.
-           If true, will raise a Validation error if `sweep_config` does not conform to
-           the schema. If false, will attempt to run the sweep with an unvalidated schema.
-
-    Returns:
-        The suggested run.
-    """
```

Comment:
these docs are valuable and we should be sure to keep them. why delete them here but not replace them in the rest of the PR?